admin-dev/themes/new-theme/.webpack/dev.js

```diff
@@ -8,28 +7,12 @@ const LiveReloadPlugin = require('webpack-livereload-plugin');
  * @param {String} hostname Development host name, sent as a parameter to webpack. Defaults to localhost
  */
 function devConfig(hostname) {
```

Comment:
If live reload is removed, then there's no need for `hostname` anymore. @PierreRambaud you couldn't get it to run alongside the css extraction plugin?

Reply:
Nothing was working for me for with Live reload. I remove unneeded variables.